make/addons.mk

```
@@ -3,9 +3,11 @@
 
 export CALICO_VERSION := v3.26.1
 export NODE_FEATURE_DISCOVERY_VERSION := 0.14.1
+export AWS_CSI_SNAPSHOT_CONTROLLER_VERSION := v6.3.0
+export AWS_EBS_CSI_VERSION := release-1.23
 
 .PHONY: addons.sync
-addons.sync: $(addprefix update-addon.,calico nfd)
+addons.sync: $(addprefix update-addon.,calico nfd aws-csi)
```

This needs to match `update-addon.aws-ebs-csi` right?
```suggestion
addons.sync: $(addprefix update-addon.,calico nfd aws-ebs-csi)
```

---

Oh I see either line 20 or 21 needs to change to match. (I would prefer `aws-ebs-csi` that matches the rest of the code)